api/v1alpha3/gcpmachine_types.go

```diff
@@ -77,6 +84,14 @@ type GCPMachineSpec struct {
 	// +optional
 	RootDeviceSize int64 `json:"rootDeviceSize,omitempty"`
 
+	// RootDeviceType is the type of the root volume.
+	// Supported types of root volumes:
+	// 1. "pd-standard" - Standrad (HDD) persistent disk
```

Comment:
small typo here: "Standrad". looks like it also got copy/pasted in a few other places below.

Reply:
Thanks, fixed!